### PR TITLE
Feature: creating inventory ingredient endpoints

### PR DESCRIPTION
**Pull request by eapg**
* adding endpoints for inventory ingredient controller
* adding e2e test for inventory ingredient controller endpoints

### CODE REVIEW

File: src/constants/http_status_code.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Status(Enum):
```
Comment (reynoldmorel) [inline src/constants/http_status_code.py:4]: Rename to HttpStatus

File: src/constants/http_status_code.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    HTTP_200_OK = 200
```
Comment (reynoldmorel) [inline src/constants/http_status_code.py:5]: Rename to OK

File: src/constants/http_status_code.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    HTTP_201_CREATED = 201
```
Comment (reynoldmorel) [inline src/constants/http_status_code.py:6]: Rename to CREATED

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/constants/http_status_code.py (hunk 1 above)
Comment (eapg) [reply src/constants/http_status_code.py:4]: done.

File: src/constants/http_status_code.py (hunk 2 above)
Comment (eapg) [reply src/constants/http_status_code.py:5]: done.

File: src/constants/http_status_code.py (hunk 3 above)
Comment (eapg) [reply src/constants/http_status_code.py:6]: done.

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**